torch/onnx/symbolic_opset9.py

```diff
@@ -719,6 +719,9 @@ def prelu(g, self, weight):
     self_rank = sym_help._get_tensor_rank(self)
     if self_rank is not None and self_rank > 2:
         weight = sym_help._unsqueeze_helper(g, weight, list(range(1, self_rank - 1)))
+    elif self_rank is not None and self_rank == 0:
+        self = sym_help._unsqueeze_helper(g, self, [0])
+    assert sym_help._get_tensor_rank(self) >= len(sym_help._get_tensor_sizes(weight)), "slope must be unidirectionally broadcastable to input tensor"
```

Comment:
The bug needs to be reproduced using TVM but will introduce a huge testing dependency. So instead I added a rank checking for it.